README.rst

```diff
@@ -27,20 +27,28 @@ Example Usage
 
 .. code-block:: python
 
-   from adb_shell.adb_device import AdbDeviceTcp
+   from adb_shell.adb_device import AdbDeviceTcp, AdbDeviceUsb
    from adb_shell.auth.sign_pythonrsa import PythonRSASigner
 
    # Connect (no authentication necessary)
    device1 = AdbDeviceTcp('192.168.0.111', 5555, default_timeout_s=9.)
    device1.connect(auth_timeout_s=0.1)
 
    # Connect (authentication required)
-   with open('path/to/adbkey') as f:
+   with open('path/to/adbkey', 'rb') as f:
```

Comment:
I think `'rb'` isnt necessary for `PythonRSASigner`, but as long as this works it's OK. And I think the cryptography signer does require bytes, so this is probably a safer practice.